crates/sncast/tests/helpers/devnet.rs

```diff
@@ -34,7 +34,14 @@ fn start_devnet() {
     }
 
     Command::new("tests/utils/devnet/bin/starknet-devnet")
-        .args(["--port", &port, "--seed", &SEED.to_string()])
+        .args([
+            "--port",
+            &port,
+            "--seed",
+            &SEED.to_string(),
+            "--account-class",
+            "cairo1",
```

Comment:
Note: In the current version of `starknet-devnet-rs`, which supports RPC `0.7.0`, it has become the default setting. Therefore, it can be potentially removed later.

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/1862

Reply:
the support is merged to master, let's conform to that

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1851/commits/ac91f9dc315adf88880ad66d910b2c5c1d43d874